### PR TITLE
openvdb: update to 8.1.0

### PR DESCRIPTION
**Pull request by jasonliu--**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010 x86_64
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: graphics/openvdb/Portfile [hunk 1]
```diff
@@ -7,8 +7,7 @@ PortGroup               active_variants 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               boost 1.0
 
-github.setup            AcademySoftwareFoundation openvdb 8.0.1 v
-revision                0
+github.setup            AcademySoftwareFoundation openvdb 8.1.0 v
 categories              graphics
```
Comment (pmetzger) [inline graphics/openvdb/Portfile:11]: Current practice is to retain the "revision 0".

Comment (jasonliu--) [reply graphics/openvdb/Portfile:12]: Oops, that was an accident. My local copy of the Portfile blew the line away when I copied it over to my git area.

**Comment by pmetzger** [comment]
@jasonliu-- any idea why the CI failed? (I haven't looked myself.)

**Comment by jasonliu--** [comment]
It looks like the CI was failing to build blender while trying to install its dependencies.

```
2021-12-29T04:48:50.7856730Z Installing dependency (161 of 299) 'p5.28-time-local' with variants '' (requesting '') ... [OK]
2021-12-29T04:49:22.0396060Z Installing dependency (162 of 299) 'p5.28-timedate' with variants '' (requesting '') ... [OK]
2021-12-29T04:49:53.6882320Z Installing dependency (163 of 299) 'p5.28-http-date' with variants '' (requesting '') ... [OK]
2021-12-29T04:50:24.7696880Z Installing dependency (164 of 299) 'p5.28-io-html' with variants '' (requesting '') ... [OK]
2021-12-29T04:50:29.1235230Z ##[error]The operation was canceled.
2021-12-29T04:50:29.1699850Z ##[group]Run sudo find \
2021-12-29T04:50:29.1700470Z [36;1msudo find \[0m
2021-12-29T04:50:29.1700830Z [36;1m  /tmp/mpbb \[0m
2021-12-29T04:50:29.1701200Z [36;1m  -maxdepth 1 \[0m
2021-12-29T04:50:29.1701580Z [36;1m  -mindepth 1 \[0m
2021-12-29T04:50:29.1701920Z [36;1m  -type d \[0m
2021-12-29T04:50:29.1702260Z [36;1m  -print \[0m
2021-12-29T04:50:29.1702630Z [36;1m  -exec chmod -R go+rX {} \;[0m
2021-12-29T04:50:29.2224640Z shell: /bin/bash -e {0}
2021-12-29T04:50:29.2225080Z ##[endgroup]
2021-12-29T04:50:29.3261320Z /tmp/mpbb/openimageio
2021-12-29T04:50:29.3362740Z /tmp/mpbb/logs
2021-12-29T04:50:29.3395580Z /tmp/mpbb/blender
2021-12-29T04:50:29.4286800Z ##[group]Run actions/upload-artifact@v2
2021-12-29T04:50:29.4287380Z with:
2021-12-29T04:50:29.4287850Z   name: logs-macos-11.zip
2021-12-29T04:50:29.4288310Z   path: /tmp/mpbb/*/logs
2021-12-29T04:50:29.4288810Z   if-no-files-found: warn
2021-12-29T04:50:29.4289260Z ##[endgroup]
```

**Comment by pmetzger** [comment]
I assume you've tested this properly on your own machine?

**Comment by jasonliu--** [comment]
Yes, and of course they all build fine, because all of the dependencies have already been previously installed.

**Pull request merged**